query_plan_test.go

```diff
@@ -18,6 +20,77 @@ func mustNewStruct(m map[string]interface{}) *structpb.Struct {
 	}
 }
 
+func TestRenderTreeUsingTestdataPlans(t *testing.T) {
+	for _, test := range []struct {
+		title string
+		file  string
+		want  []QueryPlanRow
+	}{
+		{
+			// Original Query:
+			// SELECT s.LastName FROM (SELECT s.LastName FROM Singers AS s WHERE s.FirstName LIKE 'A%' LIMIT 3) s WHERE s.LastName LIKE 'Rich%';
+			title: "With Filter Operator",
+			file: "testdata/plans/filter.input.json",
+			want: []QueryPlanRow{
+				{
+					ID:           0,
+					Text:         "Serialize Result",
+				},
+				{
+					ID:           1,
+					Text:         "+- Filter",
+					Predicates: []string{"Condition: STARTS_WITH($LastName, 'Rich')"},
+				},
+				{
+					ID:           2,
+					Text:         "   +- Global Limit",
+				},
+				{
+					ID:           3,
+					Text:         "      +- Distributed Union",
+					Predicates: []string{"Split Range: STARTS_WITH($FirstName, 'A')"},
+				},
+				{
+					ID:           4,
+					Text:         "         +- Local Limit",
+				},
+				{
+					ID:           5,
+					Text:         "            +- Local Distributed Union",
+				},
+				{
+					ID:           6,
+					Text:         "               +- FilterScan",
+					Predicates: []string{"Seek Condition: STARTS_WITH($FirstName, 'A')"},
+				},
+				{
+					ID:           7,
+					Text:         "                  +- Index Scan (Index: SingersByFirstLastName)",
+				},
+			}},
+	} {
+		t.Run(test.title, func(t *testing.T) {
+			b, err := ioutil.ReadFile(test.file)
+			if err != nil {
+				t.Error(err)
```

Comment:
This should be `t.Fatal` as it can't continue the test.

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/87/commits/a05abe82a39ef3bfa3bf9b9100172c1a42d8cc22